### PR TITLE
updated test-result csv example and fix broken link

### PR DESCRIPTION
**Pull request by zdeveloper**
# FRONTEND PULL REQUEST

## Related Issue

- part of #4359

## Changes Proposed

- Link to corrected test results csv example
- Fix broken link in docs page

### CODE REVIEW

File: frontend/src/app/testResults/uploads/CsvSchemaDocumentation.tsx [hunk 1]
```diff
@@ -160,7 +160,7 @@ const CsvSchemaDocumentation = () => {
           <ul>
             <li>
               <a
-                href="/getting-started/testing-facilities/csv-schema"
```
Comment (zdeveloper) [inline frontend/src/app/testResults/uploads/CsvSchemaDocumentation.tsx:163]: This link led to an error page since that page doesn't exisit

**Review by nathancrtr — COMMENTED** [review]
@zdeveloper looks like another template link needs to be updated as well:

https://github.com/CDCgov/prime-simplereport/blob/9921ee32bcd06f7898a3634d29a3d65ca880ae6c/frontend/src/app/testResults/uploads/CsvSchemaDocumentation.tsx#L184-L186

**Comment by zdeveloper** [comment]
> @zdeveloper looks like another template link needs to be updated as well:
> 
> https://github.com/CDCgov/prime-simplereport/blob/9921ee32bcd06f7898a3634d29a3d65ca880ae6c/frontend/src/app/testResults/uploads/CsvSchemaDocumentation.tsx#L184-L186

Great catch! fixed the other spot and did a site wide search for the old link to make sure we don't have it anywhere else

File: frontend/src/app/testResults/uploads/CsvSchemaDocumentation.tsx (hunk 1 above)
Comment (mehansen) [reply frontend/src/app/testResults/uploads/CsvSchemaDocumentation.tsx:163]: this change can close #4225 as well

**Review by mehansen — APPROVED** [review]

**Pull request merged**